meinberlin/apps/votes/views.py

```diff
@@ -29,10 +29,11 @@ class ExportTokenDashboardView(
     template_name = "meinberlin_votes/token_export_dashboard.html"
 
     def _get_number_of_tokens(self):
-        return VotingToken.objects.filter(module=self.module, is_active=True).count()
+        tokens = VotingToken.objects.filter(module=self.module, is_active=True)
```

Comment:
Maybe a docstring """Return total and downloadable token count for module.""" or sth. like that? I mean, it is quite obvious now, but might not be next time we look at it?